doc/nrf/releases/release-notes-latest.rst

```diff
@@ -93,4 +93,11 @@ The following list summarizes the most important changes inherited from upstream
 
 The following list contains |NCS| specific additions:
 
-*
+* The Bluetooth controllers default uses split architecture
```

Comment:
What does this mean? What is "split architecture" - split between Host and Controller?
And why "controllers" - doesn't Zephyr have only one Controller?

Maybe something like:
```suggestion
* Configured the Zephyr Bluetooth stack to build only the Host or only the Controller instead of doing a combined build (see :ref:`zephyr:bluetooth-build-types`).
  The reason for this default configuration is to add support for the :ref:`ble_controller`.
```
The links point to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/bluetooth/bluetooth-arch.html#build-types and https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/ble_controller/README.html#nrf-bluetooth-le-controller , respectively.

Reply:
That should actually be described in the docs, although that section seems a bit empty:
https://docs.zephyrproject.org/latest/guides/bluetooth/bluetooth-arch.html#split

Reply:
and here:
http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/CONFIG_BT_LL_SW_SPLIT.html#cmdoption-arg-config-bt-ll-sw-split

Reply:
What is the purpose here? To describe ways that NCS is different from Zephyr? In that case I think until the SoftDevice controller is the default this paragraph can be removed.

@rugeGerritsen Any opinions here?

Reply:
Note that Legacy (as opposed "split") has been removed, so Zephyr only has one choice symbol here.

@cvinayak @carlescufi https://docs.zephyrproject.org/latest/guides/bluetooth/bluetooth-arch.html#split
This could perhaps be remove as well? Only one option now.

Reply:
> What is the purpose here? To describe ways that NCS is different from Zephyr?

The idea is to point out persistent differences between NCS's Zephyr fork and upstream Zephyr.

Reply:
Right, so remove this sentence then. Since this is not different for upstream.

Reply:
Yes, please remove this @tejlmand

Reply:
Then why do we still have this path ?
https://github.com/nrfconnect/sdk-zephyr/commit/e85c858c21752b50b5b9e6f26cf78300c168a882

Reply:
@tejlmand Because we add another option to that menu, and since order matters we need the default statement there to keep the same default.

Reply:
got it.
Thanks for explaining :)